gulpfile.babel.js

```diff
@@ -85,13 +86,42 @@ gulp.task('mergeTW', ['zhTW'], () => {
   });
 });
 
+function parseTitle() {
+  const totalTutorial = Object.keys(tutorialJSON).length;
+  let en = '';
+  let zhTW = '';
+
+  for (let i = 0; i < totalTutorial; i++) {
+    let parseEnContent = `
+      <a href="en/how-to-back-up-music.html" class="list-group-item">${tutorialJSON[i].en}</a>
+    `;
+    en += parseEnContent;
+
+    let parseTwContent = `
+      <a href="zh-TW/how-to-back-up-music.html" class="list-group-item">${tutorialJSON[i]['zh-TW']}</a>
+    `;
+    zhTW += parseTwContent;
+  }
```

Comment:
This implementation is bad because build script should not know things about client codes. The better way here is write a simple js file and make it fetch the JSON file and render them directly on the page.